acct.bionimbus.org/portal/gitops.json

```diff
@@ -44,32 +44,46 @@
       },
       "buttons": [
         {
-          "name": "Data Model",
-          "icon": "data-field-define",
-          "body": "The ACCOuNT Data Commons ingests and exports data that conform to a graph-like relational data model.",
+          "name": "Define Data Field",
```

Comment:
Nah. Needs to be "Data Model"

Reply:
Again, the automation is just copying whatever it finds on `gitops-qa/qa-acct` so... if you plan to fix this in this PR, that's ok but please remember to also fix it in gitops-qa, otherwise, in the next release cycle, when this automation is executed again, the same bad text will be copied over.